core/src/poh_recorder.rs

```diff
@@ -57,12 +59,19 @@ impl PohRecorder {
         self.poh.hash();
     }
 
+    pub fn start_slot(&self) -> u64 {
+        self.start_slot
+    }
+
     pub fn bank(&self) -> Option<Arc<Bank>> {
         self.working_bank.clone().map(|w| w.bank)
     }
     pub fn tick_height(&self) -> u64 {
         self.poh.tick_height
     }
+    pub fn ticks_per_slot(&self) -> u64 {
+        self.ticks_per_slot
+    }
     // synchronize PoH with a bank
     pub fn reset(&mut self, tick_height: u64, blockhash: Hash) {
```

Comment:
how about passing both slot and tick_height to this function instead?  might obviate the need for storing ticks_per_slot in poh_recorder